src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/custom.py

```diff
@@ -37,33 +43,56 @@ def _server_restore(cmd, client, resource_group_name, server_name, parameters, *
     except Exception as e:
         raise ValueError('Unable to get source server: {}.'.format(str(e)))
 
+def _restore_mysql(cmd, client, resource_group_name, server_name, parameters, **kwargs):
+    _server_restore(cmd, client, resource_group_name, server_name, parameters, **kwargs)
```

Comment:
Same for _create_server_for_xxx, please work with qingqing to update the swagger file, and make the create server operation id the same for mysql and pg